includes/widgets/class-urlslab-keywords-links.php

```diff
@@ -755,4 +765,9 @@ private function get_link_element_text( $dom ): string {
 
 		return '';
 	}
+
+	public static function keywords_changed() {
+		wp_cache_flush_group( 'urlslab_kw' );
```

Comment:
This function is provided in wordpress WordPress 6.2-alpha-54859, The API might change. maybe we can use this only in certain conditions.